### ISSUE TITLE
User Review

### ISSUE THREAD

**Issue by gkwan-ibm** [OpenLiberty/guide-microprofile-telemetry-jaeger#4]
Allow a person with a fresh mind to read the guide and execute the steps described in the guide, test whether each step is clear enough to understand, and no logic errors.

Review and walkthrough end-to-end by a non-SME or a new user with a critical and fresh pair of eyes:

- [x] Record the time it takes to go through the entire guide
- [x] Test each step in the guide is clear and easy to understand
- [x] Ensure all hotspots work
- [x] Ensure all URLs work and open new tab
- [x] Ensure no logic errors
- [x] Report anything that does not work
- [x] Report anything usability issues (confusing or hard to follow)
- [x] Does the guide achieve basic understanding/working knowledge of the topic in the guide?

**Comment by saumyapandyaa** [comment]
I encountered a few things that might be issues, but I'm not entirely sure. Here are my notes for you to review.

- http://localhost:9081/inventory/systems/localhost shows an error in terminal. says “Failed to export spans. The request could not be executed. Full error message: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:4317”
- http://localhost:16686/ doesn’t load. says “Unable to connect”
- Hitting ‘enter/return’ at the end prints logs and an error message that says “Failed to export spans. The request could not be executed. Full error message: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:4317”

**Comment by gkwan-ibm** [comment]
hi @saumyapandyaa 

Did you install Jaeger as the **Additional prerequisites** section described?

**Comment by saumyapandyaa** [comment]
hi
Yes I did but I guess I did it in my terminal and ran the other commands in cloud. Silly mistake! I'll retry the whole process. Thanks for pointing it out!

**Comment by saumyapandyaa** [comment]
My terminal is acting strange today. When I run the all-in-one command, I get this error message: "docker: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?" I tried searching online and attempted a few solutions, but it seems like something is going wrong. Any suggestions are welcome!

**Comment by gkwan-ibm** [comment]
hi @saumyapandyaa, you can download and run the Jaeger binary if you cannot resolve the docker issue

**Comment by saumyapandyaa** [comment]
I managed to go through the guide in about 20-30 minutes, as long as there were no issues with Docker. It was pretty doable and easy to follow, and I had no problem understanding the instructions. I've also ticked off all the boxes you mentioned in the issue.

**Comment by gkwan-ibm** [comment]
guide issue - https://github.com/OpenLiberty/guides-common/issues/888
